packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -424,6 +424,7 @@ def vip_workload_test(dcos_api_session, container, vip_net, proxy_net, ipv6,
     return (vip, hosts, cmd, origin_app, proxy_app, pm_app)
 
 
+@pytest.mark.xfail(reason="DCOS-53552")
```

Comment:
Hi @cprovencher  - this should be in this format

```
@pytest.mark.xfailflake(
    jira='DCOS-53552',
    reason=' description.',
    since='2019-07-11'
)
```